src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+
+import {
+  listenToEvent, callApi, displayDownwards,
+} from './use.js';
+
+import {
+  url, getScores,
+} from './storage.js';
+
+const root = document.querySelector('.root');
+
+const handleDisplay = async (container, apiScore, display, api, link) => {
+  const scores = await apiScore(api, link);
+  return display(scores, container);
+};
+handleDisplay(root, getScores, displayDownwards, callApi, url);
+
+const refresh = (container, handleDisplay, apiScore, display, api, link) => {
+  let callHandleDisplay;
+  let emptyValue;
+  listenToEvent('click', (event) => {
+    const { action } = event.target.dataset;
+    if (action === 'refresh') {
+      callHandleDisplay = handleDisplay(container, apiScore, display, api, link);
+    } else if (action === 'submit') {
+      const input = document.querySelectorAll('input');
+      input.forEach((currentItem) => {
+        currentItem.value = '';
+        emptyValue = currentItem.value;
+      });
+    }
+  });
+  return callHandleDisplay || emptyValue;
```

Comment:
- [ ] When clicking the submit button it seems to be submitting the form twice in a row, please consider taking the time to fix it if you would like to make your application work better and more presentable. 👍

Reply:
Hello Mike, thanks for the review. I will make sure I fix it. I switched it to functional programming and used some of its principles. 
I would appreciate links to some resources on functional programming if you have them.